README.md

```diff
@@ -1 +1,147 @@
 # Awesome-Books-with-Modules
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Awesome-Books-with-Modules] <a name="about-project"></a>
+**[Awesome-Books-with-Modules]** is the first project at Microverse that is meant to test one's knowledge on gitflow and Es Linter Configuration. This project was build using Html and css. Git, Github, and Vs Code Studio were key technological tools used in this project.
```

Comment:
- [ ] [OPTIONAL] I think you can provide a better details for your project, this is for the users who checks out this project of your so try to think fro their view too, the description should make sense to non-technicalpeople too. you can use something like "This is a project to help you find and remember the best books you read. You can add favourite books, or books you'd like to read, you can create a book collection and this information will be stored and saved for you even if the browser is closed, you can access your collection anytime anywhere, a must have app with easy UI/UX."